app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), data:{turbo:false}) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <% posts_counter: 0 %>
+
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
```

Comment:
- [ ] I keep getting the following error when try to sign up. :point_down: 
![Screenshot from 2022-05-09 22-07-45](https://user-images.githubusercontent.com/67087664/167462032-d17aa00d-b3b2-43a4-b96d-1afffe269c67.png)
Please fix it.